src/main/java/duke/Date.java

```diff
@@ -0,0 +1,18 @@
+package duke;
+
+import java.time.LocalDate;
+
+public class Date {
+    static String formatedDate;
+
+    public static String toString(LocalDate str) {
```

Comment:
Maybe you can use DateTimeFormatter.ofPattern(pattern) method to make this code shorter